app/src/main/java/org/wikipedia/analytics/NotificationsABCTestFunnel.kt

```diff
@@ -0,0 +1,39 @@
+package org.wikipedia.analytics
+
+import org.wikipedia.WikipediaApp
+
+class NotificationsABCTestFunnel :
+    Funnel(WikipediaApp.getInstance(), SCHEMA_NAME, REV_ID, SAMPLE_LOG_ALL) {
+    private val abTestImpl = ABTestFunnel("notificationIcons", ABTestFunnel.GROUP_SIZE_3)
+
+    val aBTestGroup: Int get() = abTestImpl.aBTestGroup
+
+    fun logShow() {
+        val item = when (aBTestGroup) {
+            0 -> "inbox"
+            1 -> "bell"
+            else -> "more"
+        }
+        log(
+            "action", "show",
+            "menuItem", item
```

Comment:
Would it be possible to have these strings as `const` variables?

Reply:
sure!